### PR TITLE
Update project setup instructions for linux users

### PR DESCRIPTION
**Pull request by Mutugiii**
- [x] I have read [Wikonnects contributing guidelines](https://github.com/tunapanda/wikonnect/blob/master/CONTRIBUTING.md).
- [x] My pull request has a descriptive title (not a vague title like `Update README.md`).
- [x] My pull request targets the `master` branch of Wikonnect.
- [x] My pull request passes all test.

Closes #XXXXX

### CODE REVIEW

**Comment by Mutugiii** [comment]
Update the project setup instructions for linux systems

File: README.md [hunk 1]
```diff
@@ -78,7 +84,26 @@ development: {
...
+Also ensure that you have redis-server running locally in your pc, To install it in your pc run the command below
+
+```
+sudo apt-get install redis-server
+```
+
+To confirm that the service is active, enter the command below
+
+```
+sudo service redis-server status
```
Comment (profnandaa) [inline README.md]: Can add a statement for to for starting the server if it's not running:
```
sudo service redis-server start
```

File: README.md [hunk 2]
```diff
@@ -78,7 +84,26 @@ development: {
...
 
-Running `knex migrate:latest ` in the `server/` directory will use the migration files in `server/migrations` to create and format the tables so that they will work with wikonnect.
+Ensure that knex is installed globally to run some of the commands below
+Using npm:
+
+```
+npm install -g knex
+```
+
+Also ensure that you have redis-server running locally in your pc, To install it in your pc run the command below
```
Comment (profnandaa) [inline README.md]: You could provide link for MacOS and Windows, may be just https://redis.io/download

**Review by profnandaa — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (Mutugiii) [reply README.md]: sure thing

File: README.md (hunk 2 above)
Comment (Mutugiii) [reply README.md]: sure

**Pull request merged**